src/server/beego/v2.0/registry/handler/catalog.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Project Harbor Authors
```

Comment:
How about the following structure for the registry handlers & the "registry"?

```go
/src
    /server
        /beego
           /v2.0
               /registry
                   /tag
                       handler.go // handler for tag
                   /catalog
                       handler.go  // handler for catalog
                   handler.go // root handler bound to the server
```
??

or

```go
/src
    /server
         /middleware
         /handlers
             /tag
                 handler.go // handler for tag
             /catalog
                 handler.go // handler for catalog
        /beego
           /v2.0
               /registry
                   handler.go // root handler bound to the server for handling "/v2"
```

Because the handlers we programmed are not related to the beego server or other API servers.

What do you think?